tools/build.py

```diff
@@ -498,6 +500,10 @@ def build_libjerry(option):
         cmake_opt.append('-DEXTERNAL_MEM_HEAP_SIZE_KB=' +
                          str(option.jerry_heaplimit))
 
+    # --jerry-heap-section
+    if option.jerry_heap_section:
+        cmake_opt.append('-DJERRY_HEAP_SECTION_ATTR=\".sram_section\"')
```

Comment:
Why is the value of the section hardcoded? Wouldn't it be better to have it coming as an argument?